scripts/linux/pkg/deb/assets/DEBIAN/prerm

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# prerm script for ravendb
+#
+# see: dh_installdeb(1)
+
+set -e
+
+if [ ! -z "$_DEB_DEBUG" ]; then
+    set -x
+fi
+
+# summary of how this script can be called:
+#        * <prerm> `remove'
+#        * <old-prerm> `upgrade' <new-version>
+#        * <new-prerm> `failed-upgrade' <old-version>
+#        * <conflictor's-prerm> `remove' `in-favour' <package> <new-version>
+#        * <deconfigured's-prerm> `deconfigure' `in-favour'
+#          <package-being-installed> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    remove|upgrade|deconfigure)
+    ;;
+
+    failed-upgrade)
+    ;;
+
+    *)
+        echo "prerm called with unknown argument \`$1'" >&2
+        exit 1
+    ;;
+esac
+
+
+if command -v systemctl &> /dev/null && systemctl cat -- ravendb.service &> /dev/null; then
+    systemctl disable ravendb
+    systemctl stop ravendb
+fi
+
+# 4.2 only
+rm -vf /usr/local/ravendb/server/librvnpal.so
```

Comment:
zstd & sodium as well
But aren't we removing the directory entirely?

Reply:
`apt` lists the files in the package and removes them one by one. Then if the dir is empty removes that dir. If something's copied manually it will prevent directory removal - so we need to remove manually the stuff we manually added.